app/app.py

```diff
@@ -31,5 +86,44 @@ def get_prices(list):
         print(dict["dataset"]["data"][0][1])
 
 
+
+@app.route('/login', methods=['GET', 'POST'])
+def login():
+    form = LoginForm()
+    if form.validate_on_submit():
+        user = User.query.filter_by(username=form.username.data).first()
+        if user:
+            if bcrypt.check_password_hash(user.password, form.password.data):
+                login_user(user)
+                return redirect(url_for('data_summary.html'))
```

Comment:
This appears to be broken -- it appears to be fixed if you change to 'data_summary_page' rather than 'data_summary.html'

Reply:
You're also going to need to add 'GET' as a method for the routing for data_summary_page, like this: 
![Screen Shot 2022-07-16 at 2 34 35 PM](https://user-images.githubusercontent.com/68709672/179367875-0465e619-7f60-4a25-8afd-d57c7a4fea59.png)